src/library/simplified/extension/marketingABTests.js

```diff
@@ -0,0 +1,13 @@
+var abTests = utag_data.context.tests;
```

Comment:
Please make sure an automation test case in Tag Management is created for to check for marketingABTests existent and it is not empty/null.